cli/command/cli.go

```diff
@@ -27,26 +27,28 @@ import (
 
 // Streams is an interface which exposes the standard input and output streams
 type Streams interface {
-	In() *InStream
-	Out() *OutStream
+	In() InputStreamer
+	Out() OutputStreamer
 	Err() io.Writer
 }
 
 // Cli represents the docker command line client.
 type Cli interface {
 	Client() client.APIClient
-	Out() *OutStream
+	Out() OutputStreamer
```

Comment:
@dnephin I did what you propose, but I can't find a way to import a function from a `*_test` file in package `cli/command` into another `*_test` file in package `cli/command/image`. From what I've been reading it seems not possible, but maybe I'm missing something.